lib/web_ui/lib/src/engine/compositor/image.dart

```diff
@@ -5,15 +5,39 @@
 // @dart = 2.10
 part of engine;
 
-/// Instantiates a [ui.Codec] backed by an `SkImage` from Skia.
+/// Instantiates a [ui.Codec] backed by an `SkAnimatedImage` from Skia.
 void skiaInstantiateImageCodec(Uint8List list, Callback<ui.Codec> callback,
     [int? width, int? height, int? format, int? rowBytes]) {
-  final SkAnimatedImage skAnimatedImage = canvasKit.MakeAnimatedImageFromEncoded(list);
+  final SkAnimatedImage skAnimatedImage =
+      canvasKit.MakeAnimatedImageFromEncoded(list);
   final CkAnimatedImage animatedImage = CkAnimatedImage(skAnimatedImage);
   final CkAnimatedImageCodec codec = CkAnimatedImageCodec(animatedImage);
   callback(codec);
 }
 
+/// Instantiates a [ui.Codec] backed by an `SkAnimatedImage` from Skia after requesting from URI.
+void skiaInstantiateWebImageCodec(String src, Callback<ui.Codec> callback,
+    WebOnlyImageCodecChunkCallback? chunkCallback) {
+  if (chunkCallback != null) {
+    chunkCallback!(0, 100);
+  }
```

Comment:
nit: the above 3 lines can be reduced to `chunkCallback?.call(0, 100);`, I think.

Reply:
Fixed in `html_image_codec.dart` too.